deploy/apecloud-mysql-cluster/values.yaml

```diff
@@ -2,66 +2,58 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 3
+## @param version MySQL cluster version
+##
+version: ac-mysql-8.0.30
 
-terminationPolicy: Delete
+## @param mode MySQL cluster topology mode, standalone, replication or raftGroup
+##
+mode: standalone
 
-clusterVersionOverride: ""
-nameOverride: ""
-fullnameOverride: ""
+## @param replicas MySQL cluster replicas
+## for standalone mode, replicas can not be changed, it is always 1
+## for replication mode, replicas can not be changed, it is always 2
+## for raftGroup mode, replicas can be changed, the default value is 3
+##
+replicas: 1
 
-monitor:
-  enabled: false
+## @param cpu
+##
+cpu: 0.5
 
-resources: { }
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+## @param memory, the unit is Gi
+##
+memory: 1
```

Comment:
should also provide value params to specify requests and limits value

Reply:
existing "resource" object also allows us to config local ephemeral storage, it cannot be just removed.

https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#setting-requests-and-limits-for-local-ephemeral-storage

Reply:
As discussed with the product manager, the cluster helm chart should target database users and expose as little K8s config details as possible, so for now we don't provide the ability to specify cpu and memory limits when creating a cluster. Users can use other ways for finer-grained resource control, like using post-renderer to modify the rendered cluster objects.